Sources/AsyncHTTPClient/NIOTransportServices/TLSConfiguration.swift

```diff
@@ -0,0 +1,140 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the AsyncHTTPClient open source project
+//
+// Copyright (c) 2018-2020 Apple Inc. and the AsyncHTTPClient project authors
```

Comment:
```suggestion
// Copyright (c) 2020 Apple Inc. and the AsyncHTTPClient project authors
```

as this is a brand new file